mhs/outbound/outbound/transmission/outbound_transmission.py

```diff
@@ -65,6 +68,10 @@ def __init__(self, certs_dir: str, client_cert: str, client_key: str, ca_certs:
                                 "max_retries": self._max_retries
                                 })
 
+                logger.info("0005", "Waiting for {retry_delay} milliseconds before next request attempt.",
+                            {"retry_delay": self._retry_delay})
+                await asyncio.sleep(self._retry_delay / 1000)
```

Comment:
Optional: possibly better to use `tornado.gen.sleep`.

Reply:
As discussed offline, I can't see any particular advantage to using the tornado function over asyncio's built-in version, so I'll keep this as it is.